charts/scorpio-broker-dist/values.yaml

```diff
@@ -0,0 +1,982 @@
+## namespace for deployment
+namespace: scorpio-broker
+
+################################### SpringArgs #######################################################################
```

Comment:
Are there no cases where it makes sense to use different arguments for the different services?

Reply:
Yes there might be scenarios where it makes sense to change the individual configs but the most common scenario will be for all of them same. Because in general people will configure postgres, kafka and eureka here which should be the same for all microservices

Reply:
But with the current solution, its not possible at all.